tlsf.c

```diff
@@ -1045,8 +1045,8 @@ void tlsf_remove_pool(tlsf_t tlsf, pool_t pool)
 ** TLSF main interface.
 */
 
-#if _DEBUG
-int test_ffs_fls()
+#if defined(_DEBUG) && _DEBUG

```

Comment:
What about adding the following after #include "tlsf.h" instead:
#if !defined (_DEBUG)
#define _DEBUG 0
#endif

Reply:
Yes, it could fix this way.

Reply:
Done.